ldb/models.py

```diff
@@ -96,15 +96,15 @@ class Organization(Entity):
     class Meta:
         verbose_name = _('organization')
         verbose_name_plural = _('organizations')
+        manager_inheritance_from_future = True
 
     name_prefix = models.CharField(_('name prefix'), max_length=100, blank=True)
     name = models.CharField(_('name'), max_length=100)
     name_short = models.CharField(_('name short'), max_length=100, blank=True)
     salutation = models.CharField(_('salutation'), max_length=100)
 
-    @models.permalink
     def get_absolute_url(self):
-        return 'ldb_organizations_detail', [str(self.id)]
+        return reverse('ldb_organizations_detail', args=[str(self.id)])
```

Comment:
Hmmm... why the `str()`?

Reply:
Because it was already there, I presume. Do we want to change it in this PR?

Reply:
Nope